src/OneExplorer/OneExplorer.ts

```diff
@@ -463,7 +463,10 @@ export class OneNode extends vscode.TreeItem {
     //
     // However, resourceExtname returns info of vscode Explorer view (not of OneExplorer).
     //    "when": "view == OneExplorerView && viewItem == config"
-    this.contextValue = node.typeAsString;
+    //
+    // To resolve above the issue, it appends the extname to the contextValue.
+    const extname = path.extname(node.uri.fsPath);
+    this.contextValue = node.typeAsString + extname;
```

Comment:
```suggestion
    // NOTE
    // this.contextValue's having a format of {NodeType}.{Ext}
    // EXAMPLE
    //   directory.
    //   basemodel.tflite
    //   product.tvn
    this.contextValue = node.typeAsString + extname;
```

Reply:
But I wonder what 'extname' would be within directory nodes.

Reply:
If it is a directory node, there is an empty string in the extname.

Reply:
How about adding the comment? It makes the format readable at a glance.

Reply:
I added the comment on the `extname` line.

Reply:
Oh sure. Thanks :-D

Reply:
Oh sure. Thanks :-D